### PR TITLE
Aws i3 support

### PR DESCRIPTION
**Pull request by yuyantingzero**

### CODE REVIEW

File: perfkitbenchmarker/providers/aws/aws_virtual_machine.py [hunk 1]
```diff
@@ -294,6 +296,10 @@ def __init__(self, vm_spec):
       raise ValueError(
           'In order to use dedicated hosts, you must specify an availability '
           'zone, not a region ("zone" was %s).' % self.zone)
+    if self.machine_type[:2].lower() == 'i3' and not FLAGS['image'].present:
```
Comment (ehankland) [inline perfkitbenchmarker/providers/aws/aws_virtual_machine.py]: the image can be specified through configs too

File: perfkitbenchmarker/providers/aws/aws_virtual_machine.py [hunk 2]
```diff
@@ -136,7 +137,8 @@ def GetBlockDeviceMap(machine_type, root_volume_size_gb=None,
     root_block_device['Ebs'].pop('Encrypted')
     mappings.append(root_block_device)
 
-  if machine_type in NUM_LOCAL_VOLUMES:
+  if (machine_type in NUM_LOCAL_VOLUMES and
+      not aws_disk.LocalDriveIsNvme(machine_type)):
```
Comment (ehankland) [inline perfkitbenchmarker/providers/aws/aws_virtual_machine.py]: Are you sure you don't ever need to supply a mapping for i3s?

Comment (yuyantingzero) [inline perfkitbenchmarker/providers/aws/aws_virtual_machine.py]: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html

File: perfkitbenchmarker/providers/aws/aws_virtual_machine.py (hunk 1 above)
Comment (yuyantingzero) [inline perfkitbenchmarker/providers/aws/aws_virtual_machine.py]: done.

**Comment by yuyantingzero** [comment]
@ehankland PTAL

File: perfkitbenchmarker/providers/aws/aws_virtual_machine.py (hunk 2 above)
Comment (ehankland) [inline perfkitbenchmarker/providers/aws/aws_virtual_machine.py]: cool - thanks

**Pull request merged**